galleries/examples/statistics/confidence_ellipse.py

```diff
@@ -217,6 +217,11 @@ def get_correlated_dataset(n, dependency, mu, scale):
 
 # %%
 #
+# .. tags::
+#
+#    plot-type: speciality, plot-type: scatter, component: ellipse, component: patch,
+#    domain: statistics,
```

Comment:
We have discussed this somewhere, but can't find the PR right now: I believe we wanted to use a standard formatting for better readability, and this can be
- one line for short lists, e.g.
  ```
  .. tags:: plot-type: speciality, domain: statistics
  ```
- one tag per line for longer lists
  ```suggestion
  #    plot-type: speciality
  #    plot-type: scatter
  #    component: ellipse
  #    component: patch
  #    domain: statistics
  ```

Reply:
My definition of "long" is "does not fit on a single line". What's your cut-off? 😄

Reply:
But you started a second line with `#    domain: statistics,`. To extrapolate, I could add more tags there, and when that's full, start a third line and continue.

Reply:
No problem 😁